Updated install plan approver

mertel-rh opened the pull request:
This branch contains a new version of a Kustomize Install Plan approver that acts as a template that allows for more control of operator install and upgrades when in manual approval mode and using ArgoCD.

strangiato commented:
It looks like the linting is flagging a few issues.

It looks like there are a few spelling errors but also some words that are probably missing from the [.wordlist-md](https://github.com/redhat-cop/gitops-catalog/blob/main/.wordlist-md) file there.  If you have any issues or questions with those let me know.

It looks like the yaml linting also flagged a few minor issues which should hopefully be fairly easily to resolve.

For functionality I will defer to @pittar!

mertel-rh commented:
Thanks all! I'll take a look at these items this afternoon when I have a
few minutes and try to get it cleaned up and meeting standards.

-Matt

On Fri, Sep 8, 2023 at 10:38 AM Trevor Royer ***@***.***>
wrote:

> It looks like the linting is flagging a few issues.
>
> It looks like there are a few spelling errors but also some words that are
> probably missing from the .wordlist-md
> <https://github.com/redhat-cop/gitops-catalog/blob/main/.wordlist-md>
> file there. If you have any issues or questions with those let me know.
>
> It looks like the yaml linting also flagged a few minor issues which
> should hopefully be fairly easily to resolve.
>
> For functionality I will defer to @pittar <https://github.com/pittar>!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/redhat-cop/gitops-catalog/pull/243#issuecomment-1711778803>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOQSAOV5QP72IXT4IYW34VLXZMUWNANCNFSM6AAAAAA4QO4RX4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

MATT ERTEL

Sr. Architect, RHCE

Red Hat Charleston <https://www.redhat.com/>

***@***.***  Phone: 8658980926
<https://red.ht/sig>
TRIED. TESTED. TRUSTED. <https://redhat.com/trusted>

mertel-rh commented:
I've added a commit to fix the spelling issues (looks like this will still fail because of tech terms that are being identified incorrectly)
I've also added a commit to fix the yaml lint error in the operatorgroup file. I see that another file, which I did not commit and is in another folder, is also failing so the yaml lint may continue to fail as well 
Please let me know if there is anything else I can do to ensure the code quality requirements are met.

pittar reviewed (approved):
Thanks for this update!
